doc/tutorials/acrn-secure-boot-with-grub.rst

```diff
@@ -0,0 +1,177 @@
+.. _how-to-enable-acrn-secure-boot-with-grub:
+
+Enable ACRN Secure Boot with GRUB
+#################################
+
+This document shows how to enable ACRN secure boot with GRUB including:
+
+  -  ACRN Secure Boot Sequence
+  -  Generate GPG Key
+  -  Setup Standalone GRUB EFI Binary
+  -  UEFI Secure Boot
+
+  Validation Environment:
+	Hardware Platform: INTEL WHL
+	ACRN Scenario: Industry
+	Service VM: Ubuntu 18.04
+	GRUB: 2.02
```

Comment:
We use 2.04 with a patch in our main Getting Started Guide... can we use that one too here? I would prefer that we stick to a single version if at all possible.

Reply:
@gvancuts  i tested using 2.02,  QA team will re-try these steps with 2.04, i can update it after that.  make sense?

Reply:
> @gvancuts i tested using 2.02, QA team will re-try these steps with 2.04, i can update it after that. make sense?

Do you know when they will test this? I would like them to test the version 2.04 **with** the patch we use in our GSG too. Can they do that?

Reply:
we have tested with grub 2.04, thanks